.circleci/config.yml

```diff
@@ -64,6 +64,7 @@ commands:
       - run:
           name: Install Node.JS
           command: |
+            sudo rm -rf /etc/apt/sources.list.d/heroku.list
```

Comment:
For posterity on why this was added:
https://discuss.circleci.com/t/heroku-gpg-issues-in-ubuntu-images/43834